src/analytics-library/swagger/analytics-library.yaml

```diff
@@ -0,0 +1,181 @@
+swagger: '2.0'
+info:
+  title: Analytics Library REST API
+  version: '0.1'
+  description: |
+    The Analytics Library REST API is provided by the Analytics service.
+    This API enables you to manage the libraries that are used to create N1QL for Analytics user-defined functions.
+
+    The API schemes and host URLs are as follows:
+
+    * `http://localhost:8095/`
+    * `https://localhost:18095/` (for secure access)
+
+    Note that this API is only available on the loopback interface of a node
+    running the Analytics service.
+
+produces:
+  - application/json
+
+consumes:
+  - multipart/form-data
+
+paths:
+  /analytics/library/:
```

Comment:
Is there really no way to get information about all the libraries and functions in a single scope, or all the functions in a single library? Similar to the Links REST API?

Reply:
Nope. Not yet anyway. Could definitely add it at some point without too much trouble.